### PR TITLE
Propose fix some typos in the docs

### PR DESCRIPTION
**Pull request by jeis2497052**
Are a couple of spelling fixes helpful ?

### CODE REVIEW

File: office/books_list.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ To be able to have each source{d} book available, we use an Excel sheet to keep
 1. Nginx HTTP Server - Second Edition -- Clement Nedelcu
 1. Once You’re lucky, twice you’re good -- Sarah Lacy
 1. OpenShift for Developers -- Grant Shipley & Graham Dumpleton
-1. Patrones de diseño -- Erich Gamma
+1. Patrons de diseño -- Erich Gamma
```
Comment (tsolakoua) [inline office/books_list.md]: Thanks for fixes 👍  However, ```Patrones de diseño -- Erich Gamma``` is correct as it's the title in the spanish version of the book.

Comment (jeis2497052) [reply office/books_list.md]: sorry, my spanish is awful.  were you able to fix this ?  should I make one more commit ?

Comment (jeis2497052) [reply office/books_list.md]: thanks for fixing and letting me know.

**Comment by eiso** [comment]
@jeis2497052 thank you!

**Pull request merged**